app/models/student.rb

```diff
@@ -1,4 +1,10 @@
 class Student < ApplicationRecord
+  VALID_EMAIL_REGEX  = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+  VALID_PHONE_REGEX  = /\A\([0-9]{3}\) [0-9]{3}-[0-9]{4}\z/
+  VALID_ZIP_REGEX    = /\A[0-9]{5}\z/
+  VALID_SMARTT_REGEX = /\A[0-9]{4}-[0-9]{6}\z/
+  LAST_NAME_ID_REGEX = /\A[0-9]{4,5}\z/
```

Comment:
I think the name is good enough. if you change one, you need to sync all other files.